apps/pwm_test/src/main.c

```diff
@@ -0,0 +1,71 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#include "sysinit/sysinit.h"
+#include <os/os.h>
+#include <pwm/pwm.h>
+#include <pwm_nrf52/pwm_nrf52.h>
+#include <bsp/bsp.h>
+
+static struct os_dev dev;
+int arg = 0;
+struct pwm_dev *pwm;
+static int value = 10000;
+
+int
+main(int argc, char **argv)
+{
+    sysinit();
+
+    struct nrf52_pwm_chan_cfg chan_conf = {
```

Comment:
Coding style thing; declare variables in the beginning of function, before other statements. I.e. before call to sysinit().